owtf/http/transaction.py

```diff
@@ -17,6 +17,7 @@
 except ImportError:
     from httplib import responses as response_messages
 
+from owtf.lib.general import *
```

Comment:
Wildcard import should be avoided at all cost to ease debugging and prevent scope override.

Reply:
+1